### ISSUE TITLE
weighted dice coefficient is zero trained for 50 epochs

### ISSUE THREAD

**Issue by sneh-debug** [ellisdg/3DUnetCNN#259]
![loss_graph](https://user-images.githubusercontent.com/57990998/90971951-6f7fe680-e532-11ea-886e-5b420eded657.png)
hi @ellisdg i am trying with weighted dice coefficient and till 50 epochs i got something weird training and validation loss.I have attached image also. Please have a look.
